var/spack/repos/builtin/packages/bedtools2/package.py

```diff
@@ -23,6 +23,7 @@ class Bedtools2(Package):
     version('2.23.0', sha256='9dacaa561d11ce9835d1d51e5aeb092bcbe117b7119663ec9a671abac6a36056')
 
     depends_on('zlib')
+    depends_on('bzip2')
```

Comment:
Does this dependency apply to all versions of the software or from a specific point on?